src/theme/Admonition/Types.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import DefaultAdmonitionTypes from '@theme-original/Admonition/Types';
+import BusinessIcon from '@mui/icons-material/Business';
+
+import './types.css';
```

Comment:
If we're going to add a separate CSS file for this component, can it be scoped to just this component? (See [src/theme/Footer/Logo/index.js](https://github.com/pomerium/documentation/blob/e8fd25bef95706c125c255b01d9fa7baa265a49a/src/theme/Footer/Logo/index.js#L10) for an example of using a CSS module.)